lucene/queries/src/java/org/apache/lucene/queries/function/FunctionValues.java

```diff
@@ -41,6 +41,9 @@
 //   want the Query carrying around big objects
 public abstract class FunctionValues {
 
+  // Default cost for FunctionValues -- ideally should be overriden by concrete implementations
+  public static final int DEFAULT_COST = 100;
```

Comment:
This is fine with me but FWIW I wouldn't even bother defining it.  It has no value set aside like this; I doubt any user code would want to refer to it.  If we want to document what the default cost is, we should say so in cost()'s javadoc.  I know some devs like to make static constants for everything but IMO it's sometimes wasted ceremony.

Reply:
Fixed, thanks